packages/base-controller/src/BaseControllerV2.ts

```diff
@@ -1,11 +1,11 @@
+import type { Json } from '@metamask/utils';
+// eslint-disable-next-line import/order
 import { enablePatches, produceWithPatches, applyPatches } from 'immer';
 
 // Imported separately because only the type is used
 // eslint-disable-next-line no-duplicate-imports
 import type { Draft, Patch } from 'immer';
```

Comment:
I don't think we need
```suggestion
import { enablePatches, produceWithPatches, applyPatches } from 'immer';
import type { Draft, Patch } from 'immer';
```